Library/docs/source/learn/users/management.md

```diff
@@ -0,0 +1,302 @@
+
+  * The `DisplayName` is added to specify the name others will see.
+
+  * The `Email` and `Password` are included to set up login credentials.
+
+```
+if (UserName.HasValue()) {
+	Request->SetUserName(*UserName);
+}
+
+Request->SetDisplayName(*DisplayName);
+Request->SetEmail(Email);
+Request->SetPassword(Password);
+```
+
+3\. **Add Optional Settings (Newsletter Subscription, Age Verification, Redirect URL, Invite Token)**
+
+* Additional optional settings may be added by CSP (if specified by the client application), such as whether the user will receive newsletters, if their age has been verified, or if they should be redirected to a specific URL after account creation.
+
+```
+auto InitialSettings = std::make_shared<InitialSettingsDto>();
+InitialSettings->SetContext("UserSettings");
+InitialSettings->SetSettings({{"Newsletter", ReceiveNewsletter ? "true" : "false"}});
+Request->SetInitialSettings({InitialSettings});
+```
+
+4\. **Invoke the API to Create the User Profile**
+
+* Finally, CSP will invoke the services endpoint to create the user profile, passing the `CreateUserRequest` object, and a callback to handle the response.
+
+```
+static_cast<ProfileApi*>(ProfileAPI)->apiV1UsersPost(Request, ResponseHandler);
+```
+
+Here is a simplified example of the CreateUser method in action:
+
+```
+void UserSystem::CreateUser(const Optional<String>& UserName,
+                            const Optional<String>& DisplayName,
+                            const String& Email,
+                            const String& Password,
+                            bool ReceiveNewsletter,
+                            bool HasVerifiedAge,
+                            const Optional<String>& RedirectUrl,
+                            const Optional<String>& InviteToken,
+                            ProfileResultCallback Callback)
+{
+	auto Request = std::make_shared<CreateUserRequest>();
+	
+	if (UserName.HasValue()) {
+		Request->SetUserName(*UserName);
+	}
+	
+	Request->SetDisplayName(*DisplayName);
+	Request->SetEmail(Email);
+	Request->SetPassword(Password);
+
+	auto InitialSettings = std::make_shared<InitialSettingsDto>();
+	InitialSettings->SetContext("UserSettings");
+	InitialSettings->SetSettings({{"Newsletter", ReceiveNewsletter ? "true" : "false"}});
+	Request->SetInitialSettings({InitialSettings});
+
+	static_cast<ProfileApi*>(ProfileAPI)->apiV1UsersPost(Request, ResponseHandler);
+}
+```
+
+
+
+
+
+
+
+
+
+## Resetting Password
+
+Password management is a critical part of securing user access in CSP. Strong password policies help protect user accounts from unauthorized access. 
+
+However, users may forget their passwords or need to reset them if they suspect a breach. In these situations, a secure password reset process is essential. It allows users to regain access to their accounts without compromising security, ensuring that passwords remain private and protected from unauthorized parties.
```

Comment:
[question] do users have several passwords and accounts each?? [nit] if not, remove plural